site/content/contribute/webapp/developer-setup.md

```diff
@@ -14,7 +14,7 @@ Set up your development environment for building, running, and testing the Matte
  - On Mac, use [Homebrew](https://brew.sh/) to install Node.js v10 and libpng:
 
     ```sh
-    brew install node@10 libpng
+    brew install node@10 libpng npm
```

Comment:
I believe npm is included in node installation, isn't it?

Reply:
I thought the same but when I run `npm run test` it throws: `env: node: No such file or directory in mac`
check this one: https://stackoverflow.com/questions/43465086/env-node-no-such-file-or-directory-in-mac

Reply:
@angeloskyratzakos npm is included with node. I think it must have been an issue with npm locally for you. Can you confirm that a fresh install of node causes this issue?

Reply:
I tried to replicate the error but I cannot anymore. So probably it was only on my machine. closing the PR